README.md

```diff
@@ -34,13 +34,19 @@
 
 ## Environment Variables
 
+A `.env` file is required to run this application. Use the `.env.sample` file
+to create a `.env` file with the desired configuration.
+
 Key | Description | Example
 --- | --- | ---
 `RUST_LOG` | Log level | `warn,info,error,debug`
 `POSTGRES_USER` | PostgreSQL User | N/A
 `POSTGRES_PASSWORD` | PostgreSQL Password | N/A
 `POSTGRES_DB` | PostgreSQL Database Name | N/A
 `POSTGRES_HOST` | PostgreSQL Hostname | N/A
+`POSTGRES_CONN_TIMEOUT` | PostgreSQL connection timeout in seconds | 2
+`DATABASE_URL` | Database URL used by SQLx | `postgres://user:secret@127.0.0.1/dbname`
```

Comment:
sqlx uses DATABASE_URL and the POSTGRES docker container uses the other ones. I think that we can use https://github.com/dotenv-rs/dotenv#variable-substitution to construct DATABASE_URL from the  POSTGRES_ by default in the .env.sample file.

cc @EstebanBorai

Reply:
+1 to build DATABASE_URL from POSTGRES variables

Reply:
If it works with SQLx as its working now Im in!

Reply:
@morenol is right here, this suggestion is out of scope.

Also the definition for the entity was done before at the definition of the problem.

We use `surname` because you will need to search for users Name and Surname separately to improve search results.

You can have:

- Email
- Username
- Name
- Surname

To find profiles around the app.